webapp/src/components/post_type/component.tsx

```diff
@@ -49,59 +49,62 @@ const PostType = ({post, connectedID, hasCall, pictures, profiles, showSwitchCal
     );
 
     return (
-        <Main data-testid={'call-thread'}>
-            <SubMain ended={Boolean(post.props.end_at)}>
-                <Left>
-                    <CallIndicator ended={Boolean(post.props.end_at)}>
-                        {!post.props.end_at &&
-                            <ActiveCallIcon
-                                fill='var(--center-channel-bg)'
-                                style={{width: '100%', height: '100%'}}
-                            />
+        <>
+            <Message>{post.message}</Message>
```

Comment:
This may need to by styled a bit better. Please tell me some magical css incantations :)